core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -46,31 +48,31 @@ object Logging {
 
   def make[R](
     logger: LogWriter[R],
-    rootLoggerName: Option[String] = None
+    rootLoggerName: Option[String] = None,
+    initialContext: LogContext = LogContext.empty
```

Comment:
I have mix fillings about that because root logger in technically annotation. so someone could pass `LogContext` with `Name` annotation and `rootLoggerName` and result fill confusing. I would keep only `initialContext` 
```
def make[R](logger: LogWriter[R], initialContext: LogContext = LogContext.empty): URLayer[R, Logging] = ...

def make[R](logger: LogWriter[R], rootLoggerName: Option[String] = None): URLayer[R, Logging] = 
  make(logger, rooLoggerName.map(name => LogContext.empty.annotate(LogAnnotation.Name, name).getOrElse(LogContext.empty))
`